release_util_script_info.txt

```diff
@@ -0,0 +1,9 @@
+#------------------------------------------------------------
```

Comment:
Note the script names have changed (and if this could be a `.md` [markdown] file even better!)